### PR TITLE
Move spirv-tools build from third_party/Android.mk

### PR DESCRIPTION
**Pull request by atgoo**
CL adding Android.mk to spirv-tools is pending.

### CODE REVIEW

**Comment by dneto0** [comment]
Did you verify that an Android-based build works?

**Comment by atgoo** [comment]
I can build libs/arm64-v8a/libshaderc_test.so together with this change https://github.com/KhronosGroup/SPIRV-Tools/pull/861 by running ndk-build from the command line.

**Comment by antiagainst** [comment]
So the next step would be to move the configuration related to glslang to glslang.

**Comment by atgoo** [comment]
OS X build failure is unrelated to the CL.

File: third_party/Android.mk [hunk 1]
```diff
@@ -104,218 +104,4 @@ LOCAL_C_INCLUDES:=$(GLSLANG_LOCAL_PATH) \
...
-LOCAL_C_INCLUDES := \
-		$(SPVTOOLS_LOCAL_PATH)/include \
-		$(SPVTOOLS_LOCAL_PATH)/source \
-		$(SPVTOOLS_LOCAL_PATH)/external/spirv-headers/include \
-		$(SPVTOOLS_OUT_PATH)
-LOCAL_CXXFLAGS:=-std=c++11 -fno-exceptions -fno-rtti -Werror
-LOCAL_STATIC_LIBRARIES:=SPIRV-Tools
-LOCAL_SRC_FILES:= $(SPVTOOLS_OPT_SRC_FILES)
-include $(BUILD_STATIC_LIBRARY)
+include $(THIRD_PARTY_PATH)/spirv-tools/Android.mk
```
Comment (dneto0) [inline third_party/Android.mk]: Use SPVTOOLS_LOCAL_PATH to find that Android.mk file.

We support placing SPIRV-Tools in places other than under third_party/spirv-tools

**Comment by dneto0** [comment]
Rebased and pushed into master as  bcadbc1

**Pull request closed without merging**